### DISCUSSION TITLE
Is Auto logout an intended behavior ?

### DISCUSSION THREAD

**Discussion by MmeCromatique** [CastagnaIT/plugin.video.netflix#1026] [Q&A] [answered]
Hello, I have Kodi installed on a Pi 4 with Recalbox, the add-on works well, but everytime I reboot my system, I have to relog.
I use the Authentification Key Method, and everytime I need to select the file, enter the pin code AND the password of my account.
Is this intended ? Is there at least a way to only enter the pin code ?
Thanks

**Comment by CastagnaIT** [comment, marked as answer]
no usually if happen is because something of wrong is happened or netflix itself has forced the logout

i do not know Recalbox and i do not know what type of linux use,
but a possible cause is that the UUID value provided by his linux version is a fake value, and at every reboot it change where it should usually be a fixed value.

Then if this value change at every reboot prevents the add-on from decoding saved credentials/login.

you can check the value after each reboot by type:
`cat /var/lib/dbus/machine-id`

or (depends on linux type, you can try to see both)

`cat /etc/machine-id`

from the linux terminal (if Recalbox provide it)
if the value change then the problem is this

**Comment by matheusortiz** [comment]
I have the same problem. Add the path of the Key, a field with the pin and paswword in settings of add-on could be a solution?
Thank's for the project.

**Comment by CastagnaIT** [comment]
is highly not recommended, the settings data are in clear this is a big problem with the security data leak

**Reply by matheusortiz** [reply]
What is the suggestion for the previous problem?

Another problem is that the subtitle and audio settings are not saved after the episodes are finished.

Thank's

**Reply by CastagnaIT** [reply]
> What is the suggestion for the previous problem?

i have explained what i need on my first answer on top

> Another problem is that the subtitle and audio settings are not saved after the episodes are finished

if you are using Kodi 18, you have to install Kodi 19, Kodi 18 have too limited framework
if you are using Kodi 19 you have to open an new Issue thread to check and track the problem
